sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/BranchConfigurationProvider.java

```diff
@@ -36,13 +36,13 @@
 
   public BranchConfiguration provide(@Nullable BranchConfigurationLoader loader, ProjectKey projectKey, GlobalConfiguration globalConfiguration) {
     if (branchConfiguration == null) {
-      Profiler profiler = Profiler.create(LOG).startInfo(LOG_MSG);
       if (loader == null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2583)